screencapture/activator.go

```diff
@@ -53,6 +53,37 @@ func EnableQTConfig(device IosDevice) (IosDevice, error) {
 	return device, err
 }
 
+func DisableQTConfig(device IosDevice) (IosDevice, error) {
+        usbSerial := device.SerialNumber
+        ctx := gousb.NewContext()
+        usbDevice, err := OpenDevice(ctx, device)
+        if err != nil {
+                return IosDevice{}, err
+        }
+        if !isValidIosDeviceWithActiveQTConfig(usbDevice.Desc) {
+            log.Debugf("Skipping %s because it is already deactivated", usbSerial)
+            return device, nil
+        }
+
+        confignum, _ := usbDevice.ActiveConfigNum()
+        log.Debugf("Config is active: %d, QT config is: %d", confignum, device.QTConfigIndex)
+
+        for i := 0; i < 20; i++{
```

Comment:
any specific reason you need to send it so often?
I would assume it to work once if there is no error.
Probably you have seen this not to be the case, may I ask that you add a comment to explain why a loop is needed?

Thank you :-)